libbpf-tools/cpudist.c

```diff
@@ -37,20 +37,20 @@ const char *argp_program_bug_address = "<bpf@vger.kernel.org>";
 const char argp_program_doc[] =
 "Summarize on-CPU time per task as a histogram.\n"
 "\n"
-"USAGE: cpudist [--help] [-O] [-T] [-m] [-P] [-L] [-p PID] [interval] [count]\n"
+"USAGE: cpudist [--help] [-O] [-T] [-M] [-P] [-L] [-p PID] [interval] [count]\n"
 "\n"
 "EXAMPLES:\n"
 "    cpudist              # summarize on-CPU time as a histogram"
 "    cpudist -O           # summarize off-CPU time as a histogram"
 "    cpudist 1 10         # print 1 second summaries, 10 times"
-"    cpudist -mT 1        # 1s summaries, milliseconds, and timestamps"
+"    cpudist -MT 1        # 1s summaries, milliseconds, and timestamps"
```

Comment:
similarly. py version is sticking with -m. What's your rationale for making these incompatible changes?